src/bp/core/routers/bots/index.ts

```diff
@@ -281,15 +281,52 @@ export class BotsRouter extends CustomRouter {
     )
 
     this.router.post(
```

Comment:
I think it would be a good idea to extract flows route to their own routers since we're gonna add more of them. We already do that for content